guardian/grid
# IMAGEDAM-957-show deny syndication warning message

## What does this change?
As an image uploader, I want to be able to add a Deny lease without a restriction and see a warning message, so that users don't use the image without checking the lease below.
and this PR about show deny syndication leases warning message if **_`showDenySyndicationWarning`_** flag has been set to true (by default it disabled) 


 

## How should a reviewer test this change?
as permissioned user and **_`showDenySyndicationWarning`_** flag has set to true

- Upload image
- Pick rights - photographer staff
- Tick Other
- Add Name 
- Click Save (without adding a restriction)
- Click Add leases
- Deny Syndication

 Then The image has a colour warning that appears to deny syndication
 and it should not act as Deny cropping lease (not croppable for unpermissioned users)
 
 
 ## Screenshoot
 
![Screenshot 2023-05-26 at 2 48 48 AM](https://github.com/guardian/grid/assets/33189781/dfa8c275-60d8-48bd-93c9-6215612020bc)



## How can success be measured?

## Who should look at this?


## Tested? Documented?
